avaframe/runScripts/runDamBreak.py

```diff
@@ -21,34 +18,27 @@
 
 # Load settings from general configuration file
 cfgMain = cfgUtils.getGeneralConfig()
-avaDir = 'data/avaDamBreak'
-cfgMain['MAIN']['avalancheDir'] = avaDir
+avalancheDir = 'data/avaDamBreak'
+cfgMain['MAIN']['avalancheDir'] = avalancheDir
 
 # Clean input directory(ies) of old work and output files
-initProj.cleanModuleFiles(avaDir, com1DFA)
+# initProj.cleanModuleFiles(avalancheDir, com1DFA)
 
 # Start logging
-log = logUtils.initiateLogger(avaDir, logName)
+log = logUtils.initiateLogger(avalancheDir, logName)
 log.info('MAIN SCRIPT')
-log.info('Current avalanche: %s', avaDir)
-
+log.info('Current avalanche: %s', avalancheDir)
 
 # Load configuration
-damBreakCfg = os.path.join(avaDir, 'Inputs', 'damBreak_com1DFACfg.ini')
-cfg = cfgUtils.getModuleConfig(com1DFA, damBreakCfg)
-cfgGen = cfg['GENERAL']
-
-# Load flow depth from analytical solution
-hL, hR, uR, phi, xR = damBreak.damBreakSol(avaDir, cfgMain, cfg)
-xR = xR * np.cos(phi)  # projected on the horizontal plane
-dtAnalysis = cfg['DAMBREAK'].getfloat('dtStep')
-
-# call com1DFAPy to perform simulation - provide configuration file and release thickness function
-dem, plotDict, reportDictList, simDF = com1DFA.com1DFAMain(avaDir, cfgMain, cfgFile=damBreakCfg)
-
-# create dataFrame of results
-inputDir = pathlib.Path(avaDir, 'Outputs', 'com1DFA', 'peakFiles', 'timeSteps')
-dataComSolDF = fU.makeSimDF(inputDir, avaDir=avaDir)
-
-# make comparison plots
-damBreak.plotComparison(dataComSolDF, hL, xR, hR, uR, dtAnalysis, cfgMain)
+damBreakCfg = pathlib.Path(avalancheDir, 'Inputs', 'damBreak_com1DFACfg.ini')
+
+# Overwriting the sphKernelRadius and meshCellSize from the ini file with the values specified in []
+for sphKernelRadius in [5, 3]:
```

Comment:
hmm.. this is varied here as you want to vary it simultaneously with the meshcellsize right? maybe we could make a function of this and then call it?